.github/workflows/e2e-run-ui-tests.yaml

```
@@ -0,0 +1,132 @@
+        default: main
+
+jobs:
+  run-test:
+    name: "Run tests - ${{ inputs.feature_auth_required && 'auth' || 'noauth' }}, test: ${{ inputs.test_ref }}, tags: ${{ inputs.test_tags }}"
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Setup minikube
+        uses: konveyor/operator/.github/actions/start-minikube@main
+        with:
+          memory: "max"
+          cpus: "max"
+
+      - name: Checkout the operator so we can pick the install-konveyor version to use dynamically
+        uses: actions/checkout@v4
+        with:
+          repository: konveyor/operator
+          ref: ${{ inputs.install_konveyor_version }}
+          path: konveyor-operator
+
+      - name: Install konveyor using ${{ inputs.install_konveyor_version }}
+        uses: ./konveyor-operator/.github/actions/install-konveyor
+        with:
+          bundle_image: ${{ inputs.bundle_image }}
+          tackle_cr: |
+            kind: Tackle
+            apiVersion: tackle.konveyor.io/v1alpha1
+            metadata:
+              name: tackle
+            spec:
+              image_pull_policy: IfNotPresent
+              feature_auth_required: ${{ inputs.feature_auth_required }}
+
+      - name: Wait for Ingress
+        run: |
+          kubectl wait \
+            -n konveyor-tackle \
+            ingress/tackle \
+            --timeout=600s \
+            --for=jsonpath='{.status.loadBalancer.ingress[0]}'
+
+          minikube_ip=$(minikube ip)
+          echo "ingress is ready at: ${minikube_ip}"
+          echo "UI_URL=https://${minikube_ip}" >>$GITHUB_ENV
+
+      - name: "Checkout the tests (ref: ${{ inputs.test_ref }})"
+        uses: actions/checkout@v4
+        with:
+          ref: ${{ inputs.test_ref }}
+          path: tackle2-ui
+
+      - name: Run login tests using default credentials from `cypress.config.ts`
+        uses: cypress-io/github-action@v6
+        env:
+          CYPRESS_baseUrl: "${{ env.UI_URL }}"
+        with:
+          working-directory: tackle2-ui/cypress
+          spec: "e2e/tests/login.test.ts"
+
+      - name: Run UI tests (${{ inputs.test_tags }}) using default credentials from `cypress.config.ts`
+        uses: cypress-io/github-action@v6
+        env:
+          CYPRESS_INCLUDE_TAGS: "${{ inputs.test_tags }}"
+          CYPRESS_baseUrl: "${{ env.UI_URL }}"
+          CYPRESS_git_user: "fakeuser"
+          CYPRESS_git_password: "${{ secrets.GITHUB_TOKEN }}"
+          CYPRESS_git_key: "${{ secrets.GITHUB_TOKEN }}"
+        with:
+          working-directory: tackle2-ui/cypress
+
+      - uses: actions/upload-artifact@v4
+        if: failure()
+        env:
+          AUTH_KIND: ${{ inputs.feature_auth_required && 'auth' || 'noauth' }}
+          TEST_KIND: ${{ inputs.test_ref }}
+        with:
+          name: test-reports-${{ env.AUTH_KIND }}-${{ env.TEST_KIND }}
+          path: |
+            tackle2-ui/cypress/reports
+            tackle2-ui/cypress/screenshots
+            tackle2-ui/cypress/videos
```

```suggestion
            tackle2-ui/cypress/run
```

The `/run` path should cover everything.

https://github.com/konveyor/tackle2-ui/blob/29db3f929c44ff450f63f23f44a67352c2de2bfc/cypress/cypress.config.ts#L38-L40